frontend/src/App.js

```diff
@@ -34,7 +37,8 @@ const styles = {
   languageButton: {
     position: "fixed",
     right: 15,
-    top: 15
+    top: 15,
+    zIndex: 998
```

Comment:
hmm this is so strange! Can you still tab to it?

Is there a way to re-arrange the HTML so that this doesn't happen?
Or is there just too much margin/padding on something that is covering it up? 

I'm concerned with the z-index solution long term because it could mess with dialogs, etc.

Reply:
This puts it on the same level as several other components in App.js (anything with the 'fixed-top' class). I think this number keeps it on top, but behind dialogs

You can still tab to it on this branch.

Reply:
Confirmed: z-index for dialogs (at least modal dialogs) is higher (1050):
![image](https://user-images.githubusercontent.com/2746350/55483653-0625cd80-55f5-11e9-9ec1-da62460834d0.png)

Reply:
Ooo interesting about the fixed-top class. In that case, I like making use of that class a little better because it means there's not custom z indexes we have to keep track of.

This seems to work for me: https://github.com/code-for-canada/project-thundercat/compare/revert-167-revert-163-hide-main-navigation...code-for-canada:try-fixed-top?expand=1

Reply:
merged it in. Thanks!
I should have started with that...